src/Mod/Sketcher/App/planegcs/GCS.cpp

```diff
@@ -4560,20 +4584,28 @@ void System::identifyConflictingRedundantConstraints(   Algorithm alg,
                 conflictingTags.begin());
 
     // output of redundant tags
-    SET_I redundantTagsSet;
-    for (std::set<Constraint *>::iterator constr=redundant.begin();
-            constr != redundant.end(); ++constr)
+    SET_I redundantTagsSet, partiallyRedundantTagsSet;
+    for (std::set<Constraint *>::iterator constr=redundant.begin(); constr != redundant.end(); ++constr) {
         redundantTagsSet.insert((*constr)->getTag());
+        partiallyRedundantTagsSet.insert((*constr)->getTag());
+    }
+
     // remove tags represented at least in one non-redundant constraint
-    for (std::vector<Constraint *>::iterator constr=clist.begin();
-        constr != clist.end(); ++constr) {
+    for (std::vector<Constraint *>::iterator constr=clist.begin(); constr != clist.end(); ++constr)
```

Comment:
Following @vosk's suggestions, I'd make the same change here, as long as you are editing the line anyway:
 ```suggestion
    for (auto &constr : clist)
```
(and then you can lose the *s in the next line)

Reply:
idem